### ISSUE TITLE
Enable full-screen presentation mode

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#1486]
Originally reported on Google Code with ID 731

```
A common request is to have BigBlueButton display a presentation in full-screen mode,
like PowerPoint, without any other UI visible.

A scenario is a professor could have two laptops, both logged into BigBlueButton.

The first laptop is hooked to an overhead projector where they display BigBlueButton
in full-screen mode.  To the local students, it looks like a normal PowerPoint presentation.

The second laptop could be logged in "regular mode", where the presenter can glance
down and see the remote students, their web cams (if enabled), and keep track if anyone
raises their hand.

```

Reported by `ffdixon` on 2010-11-11 19:43:38

**Comment by ffdixon** [comment]
Revisiting this capability for the HTML5 client.

**Comment by antobinary** [comment]
Implemented in HTML5 client in BBB 2.2-dev circa January 2019 https://github.com/bigbluebutton/bigbluebutton/pull/6640
